doc/appendix-de-kassensichv/cash-register-integration/receipt-sequences-creation.md

```diff
@@ -0,0 +1,174 @@
+---
+slug: /poscreators/middleware-doc/germany/receipt-sequences-creation
+title: Receipt sequences creation
+---
+
+# Receipt sequences creation
+
+In the chapter [Single receipt creation](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/germany/single-receipt-creation), the creation of single receipts using either implicit and/or explicit flow has been described.
+
+In this chapter, we will describe how to connect those single receipts to receipt sequences to integrate complex business cases.
+
+## Referencing previous receipts within a queue
+
+#### Use case examples
+
+- Multiple (long lasting) transactions/orders/consumptions before payment is made (gastronomy, ...)
+- NFC-based/membership-based order solutions (e.g. accommodation/wellness, employee cards)
+- Down payments
+
+#### How to use
+
+Connect requests representing a business action with ['cbReceiptReference'](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/germany/data-structures#single-fields).
+
+#### Workflow example
+
+![referencing-previous-receipts](media/referencing-previous-receipts.png)
+Two friends are having a beer in a bar.  Because it is good German beer, they are ordering another one. They pay with one bill.
+
+#### Code examples
+
+Code examples of receipt sequences can be found in our [postman collection](https://middleware-samples.docs.fiskaltrust.cloud/#e9b0b712-2dda-4c4c-a061-16d72daa723b).
+
+## Splitting receipts
```

Comment:
should we distinguish here between a bill, a pos receipt, a order? or are all this receipts? in the example we are splitting the payment in to two pos-receipts.

Reply:
totally right, i renamed it to pos-receipt to be more specific